modules/tinymce/src/core/main/ts/autocomplete/AutocompleteLookup.ts

```diff
@@ -15,12 +15,13 @@ export interface AutocompleteLookupInfo {
   lookupData: Promise<AutocompleteLookupData[]>;
 }
 
-const isPreviousCharContent = (dom: DOMUtils, leaf: Spot.SpotPoint<Node>) =>
+const isPreviousCharContent = (dom: DOMUtils, leaf: Spot.SpotPoint<Node>) => {
   // If at the start of the range, then we need to look backwards one more place. Otherwise we just need to look at the current text
-  TextSearch.repeatLeft(dom, leaf.container, leaf.offset, (element, offset) => offset === 0 ? -1 : offset, dom.getRoot()).filter((spot) => {
+  return TextSearch.repeatLeft(dom, leaf.container, leaf.offset, (_element, offset) => offset === 0 ? -1 : offset, dom.getParent(leaf.container, dom.isBlock)).filter((spot) => {
```

Comment:
```suggestion
  // If at the start of the range, then we need to look backwards one more place. Otherwise we just need to look at the current text
  const root = dom.getParent(leaf.container, dom.isBlock);
  return TextSearch.repeatLeft(dom, leaf.container, leaf.offset, (_element, offset) => offset === 0 ? -1 : offset, root).filter((spot) => {
```
Maybe? This line is getting quite long is all.

Reply:
More than happy to.